File: templates/matlab/physics_field_interaction_matrix.m
function fig = physics_field_interaction_matrix()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('bubble_matrix', 2013, 'physics field analysis: interaction bubble matrix', 'physics field analysis', 'interaction bubble matrix');
end

File: templates/python/physics_field_interaction_matrix.py
"""physics_field_interaction_matrix: 物理场分析交互气泡矩阵（matrix 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='physics field analysis: interaction bubble matrix'):
    return make_template_figure('bubble_matrix', seed=2013, title=title, domain='physics field analysis', topic='interaction bubble matrix')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
